nixos/modules/system/boot/stage-2.nix

```
@@ -87,6 +87,15 @@ in
         '';
       };
 
+      stage2Greeting = mkOption {
+        type = types.str;
+        default = "<<< ${config.system.nixos.distroName} Stage 2 >>>";
+        defaultText = lib.literalExpression ''"<<< ''${config.system.nixos.distroName} Stage 2 >>>"'';
```

```suggestion
        defaultText = literalExpression ''"<<< ''${config.system.nixos.distroName} Stage 2 >>>"'';
```
Same as before, let's keep it consistent